tools/babel-config.js

```diff
@@ -0,0 +1,21 @@
+// This is the same set of transformations that we will enable in a future
+// official React package
+module.exports = {
```

Comment:
Updated this branch to use the external helpers provided by babel, hopefully that is good enough for a first pass.